src/components/JobsPanelDataInputs/JobsPanelDataInputs.js

```diff
@@ -33,9 +36,15 @@ const JobsPanelDataInputs = ({
     jobsPanelDataInputsReducer,
     initialState
   )
+  const pathType = inputsState.newInput.path.pathType
 
   useEffect(() => {
-    if (inputsState.newInput.path.pathType !== S3_INPUT_PATH_TYPE) {
+    if (
+      pathType !== AZURE_STORAGE_INPUT_PATH_TYPE ||
+      pathType !== GOOGLE_STORAGE_INPUT_PATH_TYPE ||
+      pathType !== S3_INPUT_PATH_TYPE ||
+      pathType !== V3IO_INPUT_PATH_TYPE
```

Comment:
Related to my comment on the PR: instead of checking that `pathType` is **not** equal to a set of values, just check that it **is** equal to the MLRun store type.
Also, it might be better to rename the `…_PATH_TYPE` part of the variable identifiers to `…_PATH_SCHEME` because this is how we call it now.